sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/WebPubSubEvent.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel.DataAnnotations;
+
+using Azure.Messaging.WebPubSub;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    /// <summary>
+    /// Output binding object to invoke rest calls to services
+    /// </summary>
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class WebPubSubEvent
```

Comment:
As mentioned in a comment about WebPubSubOperation, I haven't seen a pattern like this where we try to encapsulate potential actions to take within a model type. Instead, we generally allow binding directly to the service client, but am open to input from others. /cc @pakrym

Reply:
Another question - why wouldn't this type just live in the SDK itself (minus the Operation property). It would make things a lot more cohesive between the extension and the SDK, and it would also match the pattern we see in other messaging services where a message/event contains an envelope that has metadata, but still has a Body or Data portion that contains the actual message/event payload.

Reply:
/cc @KrzysztofCwalina regarding the SDK question